app/controllers/feeds_controller.rb

```diff
@@ -1,15 +1,36 @@
 class FeedsController < ApplicationController
   respond_to :html, :atom
+
   def index
+    urls = User.pluck(:feed_url).reject { |f| f.blank?}
```

Comment:
Space missing inside }.<br>Pass `&:blank?` as an argument to `reject` instead of a block.